types/enums/ProjectTypes.ts

```diff
@@ -73,13 +85,14 @@ export const ProjectTypes = {
   'alter.openings.add.door.side':
     'Add new doorways to the side of the building',
   'alter.openings.add.window': 'Add one or more new windows',
```

Comment:
This one is still `window` not `windows`?

Reply:
Just seen comment below about remaining singular and plural instances 👍

Reply:
This example is actually different than the singular/plural "duplicates" like "outbuilding" below - from what I found in content, it's actually always singular `window` in the parent case, and then children switch to plural `windows` :dizzy: 

`door` is always singular, so this was easy to confuse ! I'd love to do a big find&replace exercise to clean these up soon, and really pay attention to these subtle details to better align values in this particular enum.